### PR TITLE
Builder dispatchers unit tests

### PR DESCRIPTION
**Pull request by tkopczynski**
Unit tests for builder dispatchers.

Part of resolving #20784

Signed-off-by: Tomasz Kopczynski tomek@kopczynski.net.pl

### CODE REVIEW

File: builder/dockerfile/dispatchers_test.go [hunk 1]
```diff
@@ -206,3 +209,259 @@ func TestOnbuild(t *testing.T) {
 		t.Fatalf("Wrong ONBUILD command. Expected: %s, got: %s", expectedOnbuild, b.runConfig.OnBuild[0])
 	}
 }
+
+func TestWorkir(t *testing.T) {
```
Comment (vdemeester) [inline builder/dockerfile/dispatchers_test.go]: `TestWorkdir` 👼

**Comment by tkopczynski** [comment]
@vdemeester fixed :)

**Comment by LK4D4** [comment]
LGTM

**Comment by vdemeester** [comment]
LGTM 🐯

**Pull request merged**